src/doc/help.py

```diff
@@ -339,3 +341,17 @@ def help_attribute_table():
     ]
     html = html_table(title, intro, inputs, outputs, more)
     return html
+#ajoute par Rachel Goree le 30/05/2017
```

Comment:
Same here. If you want to keep it, you need to add a space between '#' and the string. Sorry, it's python syntax ;-)